tests/pds/restore_test.go

```diff
@@ -103,3 +105,195 @@ var _ = Describe("{PerformRestoreToSameCluster}", func() {
+								log.FailOnError(err, "Failed during restore.")
+								restoredDeployment, err = restoreClient.Components.DataServiceDeployment.GetDeployment(restoredModel.GetDeploymentId())
+								log.FailOnError(err, fmt.Sprintf("Failed while fetching the restore data service instance: %v", restoredModel.GetClusterResourceName()))
+								deploymentsToBeCleaned = append(deploymentsToBeCleaned, restoredDeployment)
+								log.InfoD("Restored as Deployment: %v successfully!", restoredModel.GetClusterResourceName())
+							}
+						}()
+					}
+					wg.Wait()
+					log.Infof("Restoring multiple backup job succeeded.")
+				})
+
+				Step("Delete Deployments", func() {
+					CleanupDeployments(deploymentsToBeCleaned)
+				})
+			}
+		})
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		err := bkpClient.AWSStorageClient.DeleteBucket()
+		log.FailOnError(err, "Failed while deleting the bucket")
+	})
+})
+
+var _ = Describe("{PerformSimultaneousRestoresDifferentDataService}", func() {
+	bkpTargetName = bkpTargetName + pdsbkp.RandString(8)
+	JustBeforeEach(func() {
+		StartTorpedoTest("PerformSimultaneousBackupRestoreForMultipleDeployments", "Perform multiple backup and restore simultaneously for different deployments.", pdsLabels, 0)
+		bkpClient, err = pdsbkp.InitializePdsBackup()
+		log.FailOnError(err, "Failed to initialize backup for pds.")
+		bkpTarget, err = bkpClient.CreateAwsS3BackupCredsAndTarget(tenantID, fmt.Sprintf("%v-aws", bkpTargetName), deploymentTargetID)
+		log.FailOnError(err, "Failed to create S3 backup target.")
+		log.InfoD("AWS S3 target - %v created successfully", bkpTarget.GetName())
+	})
+
+	It("Perform multiple restore within same cluster", func() {
+		stepLog := "Deploy data service and take adhoc backup, "
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			backupSupportedDataServiceNameIDMap, err = bkpClient.GetAllBackupSupportedDataServices()
+			log.FailOnError(err, "Error while fetching the backup supported ds.")
+			for _, ds := range params.DataServiceToTest {
+				_, supported := backupSupportedDataServiceNameIDMap[ds.Name]
+				if !supported {
+					log.InfoD("Data service: %v doesn't support backup, skipping...", ds.Name)
+					continue
+				}
+				stepLog = "Deploy and validate data service"
+				Step(stepLog, func() {
+					log.InfoD(stepLog)
+					dsDeployment, _, _, err := DeployandValidateDataServices(ds, params.InfraToTest.Namespace, tenantID, projectID)
+					log.FailOnError(err, "Error while deploying data services")
+
+					// TODO: Add workload generation
+
+					dsEntity = restoreBkp.DSEntity{
+						Deployment: dsDeployment,
+					}
+					log.Infof("Details DSObject- %v, Name - %v, DSEntity - %v", dsDeployment, dsDeployment.GetClusterResourceName(), dsEntity)
+					deploymentDSentityMap[dsDeployment] = dsEntity
+				})
+			}
+		})
+		stepLog = "Perform adhoc backup and validate them for all the deployed data services."
```

Comment:
same as above, need to clean up the backup jobs created in target cluster post test completes

Reply:
Automation is ready, blocked due to following bug: https://portworx.atlassian.net/browse/DS-5981